Content.Server/Tabletop/TabletopSystem.Chess.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Content.Shared.DrawDepth;
+using Robust.Server.GameObjects;
+using Robust.Shared.Map;
+using Robust.Shared.Maths;
+
+namespace Content.Server.Tabletop
+{
+    public partial class TabletopSystem
+    {
+        private void SetupChessBoard(MapId mapId)
+        {
+            var chessboard = _entityManager.SpawnEntity("ChessBoardTabletop", new MapCoordinates(-1, 0, mapId));
+
+            if (chessboard.TryGetComponent<SpriteComponent>(out var spriteComponent))
+            {
+                spriteComponent.Scale = new Vector2(1, 1);
+                spriteComponent.DrawDepth = (int) DrawDepth.FloorTiles;
+            }
```

Comment:
Please don't fiddle with sprites without going through a visualizer. We'll need them to get photography eventually.